ironfish/src/primitives/rawTransaction.ts

```diff
@@ -187,6 +187,16 @@ export class RawTransaction {
 
     return posted
   }
+
+  sender(): string | undefined {
+    if (this.spends.length > 0) {
+      return this.spends[0].note.owner()
+    } else if (this.outputs.length > 0) {
+      return this.outputs[0].note.sender()
+    } else if (this.mints.length > 0) {
+      return this.mints[0].creator
```

Comment:
This should _technically_ be `.owner`, but I see that `RawTransaction` lacks that information right now. We'll need to tackle that if/when ownership v2 comes around. No action needed here, just mentioning it